src/ResourceManager/Relay/Commands.Relay/Models/RelayNamespaceAttributesUpdateParameter.cs

```diff
@@ -40,7 +40,7 @@ public RelayNamespaceAttirbutesUpdateParameter(IDictionary<string, string> tags
             Tags = tags;
         }
 
-        public RelayNamespaceAttirbutesUpdateParameter(RelayNamespaceUpdateParameter relaynamespaceupdateparameter)
+        public RelayNamespaceAttirbutesUpdateParameter(RelayUpdateParameters relaynamespaceupdateparameter)
```

Comment:
does this new type have all the properties of the old type?  If not, is this exposed via cmdlet output or parameters?